pulumitest/run.go

```diff
@@ -93,3 +109,38 @@ func tryReadStackExport(path string) (*apitype.UntypedDeployment, error) {
 	}
 	return &stackExport, nil
 }
+
+func fixupStackName(stateFile *apitype.UntypedDeployment, newStackName string) (*apitype.UntypedDeployment, error) {
+	oldStackName, err := parseStackName(stateFile)
+	if err != nil {
+		return nil, err
+	}
+	if oldStackName == newStackName {
+		return stateFile, nil
+	}
+	newDeployment := bytes.ReplaceAll([]byte(stateFile.Deployment), []byte(oldStackName), []byte(newStackName))
```

Comment:
why not use `strings.ReplaceAll`?

Reply:
`stateFile.Deployment` is a `json.RawMessage` which is an alias for `[]byte`.